src/components/tables/TableConnected.tsx

```diff
@@ -79,14 +111,21 @@ const mapDispatchToProps = (
   onUnmount: () => {
     dispatch(removeTable(ownProps.id));
   },
-  onModifyData: (shouldResetPage: boolean, tableCompositeState: ITableCompositeState, previousTableCompositeState: ITableCompositeState) => {
+  onWillUpdate: (actions: IActionOptions[]) => {
+    if (actions.length) {
+      dispatch(addActionsToActionBar(`${ownProps.id}action-bar`, actions));
```

Comment:
whenever you want to use the action-bar id, you should use `getTableChildComponentId(ownProps.id, TableChildComponent.ActionBar)` see TableUtils for that

Reply:
you can check line 131